### DISCUSSION TITLE
Other client frameworks

### DISCUSSION THREAD

**Discussion by haarse** [amplication/amplication#1205] [General]
Are you planning to add more client frameworks, like vue?
And other server frameworks?

**Comment by yuval-hazaz** [comment]
Hi @haarse 
We focus on the back-end so - no plans to add more client frameworks. We provide the React Admin as a basic starting point to start working with the server and may add more functionality to it but only as part of moving forward on the back-end.

As for the server frameworks, we may consider in the futures add more options - but only after we provide all the core features that are currently planned
